chain/client/tests/bug_repros.rs

```diff
@@ -85,7 +85,7 @@ fn repro_1183() {
                                         to.to_string(),
                                         &InMemorySigner::from_seed(from, KeyType::ED25519, from),
                                         1,
-                                        block.hash(),
+                                        block.header.inner.prev_hash,
```

Comment:
curious what breaks if we provide the current block hash instead of the previous block hash?
seems like it should work, a transaction can in fact be sent before the receiving block producer learns about the last block the transaction initiator has seen?

Reply:
It does work just incorrectly. 
Because the current block is actually produced by one guy, but not received by the rest yet (this is code that handles network dispatch :X ) and so all tx fail except on this guy.